src/styles/layout/modal.less

```diff
@@ -31,6 +31,22 @@
     width: @modal-width-narrow;
   }
 
+  .icon-button-collection {
+
+    button {
+      height: auto;
+
+      &:last-child {
+        margin-bottom: 0;
```

Comment:
We shouldn't need this now since it's handled by the `button-collection-stacked` class.